Go 1.24 introduces a new vet check ("non-constant format string in call to
fmt.Errorf") which upstream haven't yet handled, and `go vet` is run as part of
the build: disable the class of checks which are breaking the build.
---
diff --git a/Makefile b/Makefile
index 8227ac8..add1494 100644
--- a/Makefile
+++ b/Makefile
@@ -179,7 +179,7 @@ fmt: ## Run go fmt
 .PHONY: vet
 vet: ## Run go vet
 	@echo Go vet... >&2
-	@go vet ./...
+	@go vet -printf=false ./...
 
 .PHONY: imports
 imports: $(GOIMPORTS)
